ctl/import.go

```diff
@@ -105,10 +102,26 @@ func (cmd *ImportCommand) Run(ctx context.Context) error {
 		}
 	}
 
+	// Determine the frame type in order to correctly handle the input data.
+	frameType := pilosa.DefaultFrameType
+	schema, err := cmd.Client.Schema(ctx)
+	if err != nil {
+		return errors.Wrap(err, "getting schema")
+	}
+	for _, index := range schema {
```

Comment:
why range over indexes and and frames? I think we can do schema.Indexes()[index].Frames()[frame]

Reply:
ohhhh, sorry - side note, we should maybe fix that, but that's fine, I'll approve